pkgs/applications/blockchains/trezor-suite/default.nix

```diff
@@ -7,12 +7,12 @@
 
 let
   pname = "trezor-suite";
-  version = "21.2.2";
+  version = "21.4.1";
   name = "${pname}-${version}";
 
   src = fetchurl {
     url = "https://github.com/trezor/${pname}/releases/download/v${version}/Trezor-Suite-${version}-linux-x86_64.AppImage";
-    sha256 = "0dj3azx9jvxchrpm02w6nkcis6wlnc6df04z7xc6f66fwn6r3kkw";
+    sha256 = "khkWilBDVhUrO4B+HnKC4hlSRh0ndZbGuC3f6BrCQZw=";
```

Comment:
Please don't use base64 hashes, but rather the original format.

Reply:
I got that hash after inserting 52 0's, building it, and replacing the 0's with the suggested hash. 

Can I use nix-prefetch-url to get the correct hash or should I use something else?

Reply:
nix-prefetch-url should work just fine I guess